src/index.html

```diff
@@ -3,14 +3,18 @@
 ---
 
 <header class="hero">
-  <h1 class="hero__title">
-    British Columbia Made
-    <small>Since 1987</small>
-  </h1>
-
-  <p class="hero__tagline">Premium Quality Industrial Safety Garments</p>
-
-  <a class="button button--accent" href="/products">Our Products</a>
+  <div class="title__container">
+      <h1 class="hero__title">
+         BRITISH COLUMBIA MADE
+      </h1>
+      <h3 class="hero__date">SINCE 1987</h3>
```

Comment:
By separating out the `h3` from the `h1` the doc renderer will define the it as a sub header. This is not semantically correct as it is part of the `h1`.
Keeping it as a small or any other inline wrap will make the browser/bot/reader read the entirety instead.